### DISCUSSION TITLE
Can't override defaultTimeout on CircleCI

### DISCUSSION THREAD

**Discussion by ghost** [cypress-io/cypress#18639] [CI setup]
I am having problems with cypress on CircleCI. DefaultCommandTimeout is not overriden, and tests fail because of this.

My cypress.json setting: "defaultCommandTimeout": 4000, i try to override this command in test with: Cypress.config('defaultCommandTimeout',30000) and on local stack this works as it should, on CircleCI i get failed test with: CypressError: Cypress command timeout of `4000ms` exceeded.

**Comment by ange47rm** [comment]
Hi, I have a similar (if not the same) issue. Locally, my cypress.json, is configured with  `"defaultCommandTimeout": 10000`, however, when the Circle CI pipeline runs, I get  `cy.wait()` timed out waiting `5000ms`, and one test occasionally fails. 

The fix seems to be re-running the Circle CI failed job and eventually the test passes. Not sure why the issue is with this specific test, which I've checked and double checked, but if there's a way to configure defaultCommandTimeout for Cypress Runs on Circle CI I'd like to know about it!

Many thanks in advance
